Added clips page

clementgalidie opened the pull request.

clementgalidie committed:
Added clips page

clementgalidie committed:
Added base clips loop

DeGraciaMathieu committed:
Merge branch 'master' into feature/clips-page

DeGraciaMathieu committed:
feat : added tailwind pagination on clips page

DeGraciaMathieu committed:
feat : removed game prefix

DeGraciaMathieu committed:
Changed featured clip design

DeGraciaMathieu committed:
Changed featured clip design

DeGraciaMathieu committed:
oops

DeGraciaMathieu committed:
Added style on footer links

DeGraciaMathieu committed:
i try stuff

DeGraciaMathieu committed:
changed background blur

clementgalidie committed:
Added clips grid

DeGraciaMathieu committed:
we display 12 clips per page

DeGraciaMathieu committed:
IOC ftw

DeGraciaMathieu committed:
added ShowClipControllerTest

clementgalidie committed:
Revamped pagination styles

DeGraciaMathieu committed:
Made with ☕

clementgalidie committed:
Updated clips view

clementgalidie commented on resources/views/clips.blade.php:
```diff
@@ -0,0 +1,50 @@
+                            </div>
+                        </a>
+                        <p class="absolute top-2 left-2 z-10 text-white">
+                            <span class="block bg-slate-700/80 px-2 py-1 rounded-sm">{{ $clip->duration }} secondes</span>
```
@DeGraciaMathieu est-ce qu'on aurait la possibilité de passer par `Laravel` pour afficher les secondes ? `Carbon` ?

DeGraciaMathieu replied:
J'vais voir ce que je peux faire

clementgalidie commented on resources/views/clips.blade.php:
```diff
@@ -0,0 +1,50 @@
+            <div class="grid grid-cols-1 px-4 sm:grid-cols-2 sm:gap-x-4 sm:gap-y-8 sm:px-0 lg:grid-cols-3 xl:grid-cols-4">
+                @foreach ($clips as $clip)
+                    <div class="relative">
+                        <a href="">
```
@DeGraciaMathieu j'ai ajouté un lien car j'imagine que chaque clip aura un lien qui redirigera vers la page du clip en question ?

DeGraciaMathieu replied:
Yep, c'est ca 👌

clementgalidie committed:
Updated CSS rules

clementgalidie committed:
Added search input

clementgalidie committed:
Added filters logic

DeGraciaMathieu committed:
Merge branch 'master' into feature/clips-page

clementgalidie committed:
Added filters styles

DeGraciaMathieu committed:
feat : added Algolia search

DeGraciaMathieu committed:
We cross fingers

DeGraciaMathieu committed:
Fix CI tests

clementgalidie committed:
Fixed filters JS error on other pages than /clips

clementgalidie committed:
Updated filtering

- Added click outside to close filters menu
- Fixed filters menu transition
- Added page reload on filter button click

clementgalidie committed:
Fixed typo

DeGraciaMathieu committed:
fixed phpstan errors

DeGraciaMathieu committed:
added more thumbnails

DeGraciaMathieu committed:
uniform clip display style

DeGraciaMathieu committed:
increasing defaults paginate

DeGraciaMathieu committed:
decreasing defaults

DeGraciaMathieu committed:
added lazyloading on clip iframe

DeGraciaMathieu committed:
Added empty clips search

DeGraciaMathieu committed:
added lazy loading everywhere

clementgalidie committed:
Updated clips view

- Added focus text color
- Fixed title and filters position on desktop

DeGraciaMathieu committed:
Added default clips sort

clementgalidie committed:
Added reset search input

clementgalidie committed:
Removed console log
